assets/js/vue-apps/index.js

```diff
@@ -50,13 +50,13 @@ function initInlineFeedback() {
 }
 
 export const init = () => {
-    Raven.addPlugin(RavenVue, Vue);
+    // Raven.addPlugin(RavenVue, Vue);
```

Comment:
Maybe add a `@TODO` or something to remind us not to leave this disabled...

Reply:
My bigger todo here is to find out why is suppresses error messages locally (the reason I disabled this)